dist/main.js

```diff
@@ -0,0 +1,15 @@
+$('#searchBtn').on('click' ,function(){
+    text = $('#crewSearch').val()
+    $.get(`/teams/${text}`,function(data){
+    $('#roster').empty()
+    const source = $('#roster-template').html()
+    const template = Handlebars.compile(source)
+    const newHTML = template(data)
+    $('#roster').append(newHTML) 
+    $('#crewSearch').val('')
```

Comment:
everything that is about the rendering should be in a Renderer class in a different file